kfs-core/src/main/resources/institutional-config.properties

```diff
@@ -59,11 +59,11 @@ spring.source.files=classpath:spring-kfs-imported-rice-beans.xml, \
   classpath:edu/arizona/kfs/fp/spring-fp.xml, \
```

Comment:
I don't see why this change was needed as an edu.arizona version of spring-cg.xml was already included in the list of spring source files.